.github/workflows/markdown-links.yml

```diff
@@ -12,6 +12,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: gaurav-nelson/github-action-markdown-link-check@1.0.13
+      - uses: thanethomson/github-action-markdown-link-check@611438d456810f86e2053576f7e62a433da10e0e
```

Comment:
👀

Reply:
IMO we are better off just disabling the checker on those files until it's fixed, rather than maintaining a fork.

Or if we really want to have a fork, maybe we can create one in the tendermint org rather than having it be a personal one. Not that I think you're doing anything untoward, but it looks sus to outside contributors to have random personal dependencies.
YMMV.